lib/travis/addons/config/notify.rb

```diff
@@ -49,6 +49,13 @@ def on_failure_for?(type)
             end
           end
 
+          def on_canceled_for?(type)
```

Comment:
@svenfuchs  Yes, I'm sorry, I worked on this over the holidays and completely forgot about it, when I got back to API and billing.
I also had that same question, do we need to separate and differentiate between cancel and errored?